### PR TITLE
Add ODI support in WebLogic Imagetool

### PR DESCRIPTION
**Pull request by chaitanyamoger**
Added ODI support in WebLogic Imagetool

### CODE REVIEW

**Review by ddsharpe — COMMENTED** [review]
Does the ODI install require any other installers like Java, WebLogic Server, FMW, etc?

File: imagetool/src/main/java/com/oracle/weblogic/imagetool/installer/FmwInstallerType.java [hunk 1]
```diff
@@ -80,7 +80,9 @@ public enum FmwInstallerType {
     WCS(Utils.toSet(FMW.products, AruProduct.WCS),
         InstallerType.FMW, InstallerType.WCS),
     OHS(Collections.singleton(AruProduct.OHS),
-        InstallerType.OHS)
+        InstallerType.OHS),
+    ODI(Collections.singleton(AruProduct.ODI),
```
Comment (ddsharpe) [inline imagetool/src/main/java/com/oracle/weblogic/imagetool/installer/FmwInstallerType.java:84]: According to the documentation at https://docs.oracle.com/cd/E17904_01/install.1111/e16453/install.htm#ODING285, a WebLogic Server install is required for ODI.  But, you do not have WLS or FMW as part of the installer list here.

Comment (ddsharpe) [reply imagetool/src/main/java/com/oracle/weblogic/imagetool/installer/FmwInstallerType.java:84]: No, I think ODI is the correct name.  If you add ODI on WLS, we can call that one ODI_WLS.  This one should stay ODI.

Comment (chaitanyamoger) [reply imagetool/src/main/java/com/oracle/weblogic/imagetool/installer/FmwInstallerType.java:84]: Ok. We will retain the name as it is.

**Comment by ddsharpe** [comment]
Committed PR, ready for next release.

**Pull request closed without merging**